src/localStorage.js

```diff
@@ -0,0 +1,7 @@
+export const getUserCoordinatesFromLocalStorage = () => {
+    return JSON.parse(localStorage.getItem('user_coordinates'));
```

Comment:
Might be a nitpick but i'd use a more specific name "ONEKM__user_coordinates" or something..

Reply:
I don't mind changing it but, local storage is already unique per domain, I don't see a reason to use namespacing here.